app/middlewares/validators/users.js

```diff
@@ -0,0 +1,38 @@
+const UsersSchemas = module.exports;
+
+UsersSchemas.post = {
+  name: {
+    in: ['body'],
+    notEmpty: true,
+    optional: false,
+    errorMessage: 'Name cannot be null or empty'
+  },
+  last_name: {
+    in: ['body'],
+    notEmpty: false
+  },
+  email: {
+    in: ['body'],
+    isEmail: {
+      bail: true,
+      errorMessage: 'Should be an email'
```

Comment:
These error messages must be in the  constants file, then import them where are necessary, like in this validation, and the tests.

Reply:
done